torchacc/utils/patch.py

```diff
@@ -92,3 +93,38 @@ def update_causal_mask(
             return None
 
         LlamaModel._update_causal_mask = update_causal_mask
+
+def patch_qwen():
+    '''
+    Modify the calculation of `rotary_seq_len` in `Qwen2FlashAttention2.forward` to avoid xla graph be executed
+    '''
+    import inspect
+    import transformers
+    from .logger import logger
+    from packaging import version
+    if version.parse(transformers.__version__) >= version.parse("4.37.0"):
```

Comment:
Print a warning message if the transformers version is not matched.

Reply:
Since qwen2 model is added to transformers in 4.37.0, I think it's fine not print anything when version is less than 4.37.0 ?

Reply:
Need to print some message when version >= 4.45.0 ?

Reply:
ok